app/javascript/mastodon/reducers/index.ts

```diff
@@ -50,7 +50,7 @@ const reducers = {
   meta,
   alerts,
   loadingBar: loadingBarReducer,
-  modal,
+  modalReducer,
```

Comment:
You should not change the key here, but use `modal: modalReducer`.

This would avoid changing the `modal` key everywhere this is accessed